Plash/Menus.swift

```diff
@@ -158,10 +158,18 @@ extension AppState {
 	func updateMenu() {
 		menu.removeAllItems()
 
+		menu.addCallbackItem(
```

Comment:
I think we should just hide the button when `Deactivated While on Battery"` is enabled, as it may be confusing to users that the button does not do anything.

Reply:
Implemented in https://github.com/sindresorhus/Plash/pull/131/commits/de8c457ed79d833051ed9ffcd044c3ad8704d48c. Let me know if there's a smarter or more elegant way to check for this, I couldn't find any.